src/settings/settingsSelectors.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import { createSelector } from 'reselect';
+import { getSettings } from '../selectors';
+
+export const getLocale = createSelector(getSettings, settings => settings.locale);
```

Comment:
`createSelector` creates a memoized selector. The benefit is that it will preserve the old values and not recalculate if the inputs haven't changed.

We don't want to use it for simple value-access selectors, as it only adds complexity and it makes the code a bit slower (negligibly slower, but still there is an overhead)